content/docs/dev/develop/clients/go.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Go Client
+description: Interact with TiKV using Go.
+menu:
+    "dev":
+        parent: TiKV Clients-dev
+        weight: 2
+        identifier: Go Client-dev
+---
+
+TiKV Go Client is still in the proof-of-concept stage and under development. You can track the development at [tikv/client-go](https://github.com/tikv/client-go/) repository.
+
+{{< warning >}}
+Before TiKV Go Client is officially released, it is not recommended to be used in a production environment.
```

Comment:
client-go has released version, maybe we can list some example here?

Reply:
Fixed, PTAL~